src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/DraftAdjudicationDto.kt

```diff
@@ -44,10 +44,18 @@ data class IncidentRoleDto(
 
 @ApiModel(value = "Details of an offence")
 data class OffenceDetailsDto(
-  @ApiModelProperty(value = "The offence code", notes = "This is the paragraph number relating to the offence", example = "3")
+  @ApiModelProperty(value = "The offence code", notes = "This is the unique number relating to the type of offence they have been alleged to have committed", example = "3")
```

Comment:
Don't use this DTO yet - i am going to refactor when we impl the new API call to get offfence info for the given code